src/common/tools/gen_cache.ts

```diff
@@ -14,21 +16,23 @@ DataCache will load this instead of building the expensive data at CTS runtime.
 Options:
   --help          Print this message and exit.
   --list          Print the list of output files without writing them.
-  --nth i/n       Only process every file where (file_index % n == i)
-  --validate      Check that cache should build (Tests for collisions).
+  --force         Rebuild cache even if they're up to date
```

Comment:
Note somewhere that different node versions can produce different outputs (so this may change files unnecessarily)

Reply:
Done